yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -687,7 +402,6 @@ export class L2Block {
       });
     }
   }
-
   // /**
   //  * Inspect for debugging purposes..
   //  * @param maxBufferSize - The number of bytes to be extracted from buffer.
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 410]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This commented out function is super stale now. Can you please just nuke it?

It was introduced for debugging when we started coding this thing.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/4541?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->